devices/fields/dac/f1.yaml

```diff
@@ -0,0 +1,13 @@
+_include:
+  - v1.yaml
+
+CR:
+  TSEL?:
+    Tim6Trgo: [0, Timer 6 TRGO event]
+    Tim8Trgo: [1, Timer 8 TRGO event]
```

Comment:
[rm0008](https://www.st.com/resource/en/reference_manual/rm0008-stm32f101xx-stm32f102xx-stm32f103xx-stm32f105xx-and-stm32f107xx-advanced-armbased-32bit-mcus-stmicroelectronics.pdf) table 74:

> Timer 3 TRGO event in connectivity
> line devices or
> Timer 8 TRGO in high-density and
> XL-density devices

Reply:
`f107.yaml` (connectivity line) has its own `TSEL` description.